components/bl-carousel-component/README.md

```diff
@@ -0,0 +1,68 @@
+# Carousel
+
+Carousel is a component of Backendless UI-Builder designer. This allows you to add some image and switch between them
+
+The component based on external [carousel](https://getbootstrap.com/docs/5.2/components/carousel/).
+## Usage
+
+### Styles and Settings
+
+<dl>
+<dt>Images Data</dt>
+<dd>Allows determinate data for carousel. This property must accept an array of objects in which URL is a mandatory field and title and content are optional or a JSON file.</dd>
+<dt>Height Image</dt>
+<dd>Allows determinate height for images</dd>
+<dt>Autoplay Delay (ms)</dt>
+<dd>Allows determinate delay for autoplay. Default value '5000'.</dd>
+<dt>Type Animation</dt>
+<dd>Allows select type of animation (slide, smooth). Default value 'slide'.</dd>
+<dt>Animation Duration (ms)</dt>
+<dd>Allows determinate duration for animation. Default value '600'.</dd>
+<dt>Control Buttons Visible</dt>
+<dd>Allows determinate control buttons visible or not visible. Default value 'unchecked'.</dd>
```

Comment:
Please, take a look to visibility description at any default component(e.g. Input/Block etc.)